features/addresses.feature

```diff
@@ -0,0 +1,13 @@
+Feature: The customers/{customer-id}/addresses service back-end
+    As a Customer Service Manager/ System Admin
+    I need the ability to do CRUD operations on addresses
+    So that I can maintain customer's address records
+
+Background:
+    Given the following customers
+        |    first_name    |   last_name   |         email         |   password   |   active   |                            addresses                             |
+        | FNU              | Akshama       | akshama@gmail.com     | 123456       | True       | FAstr,FActy,FAst,FAcntry,FApc                                    |
```

Comment:
each long string with no spaces in between is an address in that row, each fields of the address are delimited by ",". 
We can have multiple addreses also, just have to separate it by a space, In that data, i have put up two addreses in my row.

Reply:
> address payload would expect 6 fields: {"street":"test", "city":"test", "state":"test", "country":"test","pin_code":"test","customer_id":<customer_id>}

we wont know the customer id, so left that. We can assign any random number in the payload for customer_id and it will assign the right one while storing.